src/navigation/MainNavigator.tsx

```diff
@@ -29,9 +33,31 @@ const settingsStackTransitionPreset = Platform.select({
 
 const MainNavigator: FunctionComponent = () => {
   const { isOnboardingComplete } = useOnboardingContext()
+  const { trackScreenView } = useAnalyticsContext()
+  const navigationRef = React.useRef<NavigationContainerRef>(null)
```

Comment:
Just a small thing but how do you feel about importing the hook directly from react and avoid the `React` prepend?